src/components/itemHelper.js

```diff
@@ -155,6 +155,33 @@ export function canEditImages (user, item) {
     return itemType !== 'Timer' && itemType !== 'SeriesTimer' && canEdit(user, item) && !isLocalItem(item);
 }
 
+export function canEditSubtitles (user, item) {
+    if (item.MediaType !== 'Video') {
+        return false;
+    }
+    const itemType = item.Type;
+    if (itemType === 'Recording' && item.Status !== 'Completed') {
+        return false;
+    }
+    if (itemType === 'TvChannel'
+        || itemType === 'Program'
+        || itemType === 'Timer'
+        || itemType === 'SeriesTimer'
+        || itemType === 'UserRootFolder'
+        || itemType === 'UserView'
+    ) {
+        return false;
+    }
+    if (isLocalItem(item)) {
+        return false;
+    }
+    if (item.LocationType === 'Virtual') {
+        return false;
+    }
```

Comment:
There are enums in the sdk for the item type (`BaseItemKind`), `LocationType`, and `MediaType. It would be great if we could make use of those here.

Reply:
I made the suggested changes, but I can't find 'Timer' and 'SeriesTimer' in the sdk Enums. I have left them as string for now, but I'm open to suggestions as to what to do about that.

Also, Video can be found in `BaseItemKind.Video` and `MediaType.Video`. I used the latter but wasn't sure

https://github.com/Ch1nkara/jellyfin-web/blob/5c60f99077ce1dada3a4af10251766b677dc8e08/src/components/itemHelper.js#L163-L178

```javascript
    if (item.MediaType !== MediaType.Video) {
        return false;
    }
    const itemType = item.Type;
    if (itemType === BaseItemKind.Recording && item.Status !== RecordingStatus.Completed) {
        return false;
    }
    if (itemType === BaseItemKind.TvChannel
        || itemType === BaseItemKind.Program
        || itemType === 'Timer'
        || itemType === 'SeriesTimer'
        || itemType === BaseItemKind.UserRootFolder
        || itemType === BaseItemKind.UserView
    ) {
        return false;
    }
```